acsys/scaling/__init__.py

```diff
@@ -18,13 +18,12 @@ async def find_service(con, node=None):
         node = await con.get_name(replier)
         _log.debug('found SCALE service at node %s', node)
         return node
-    except acsys.status.Status as exception:
-        if exception == acsys.status.ACNET_REQTMO:
-            raise acsys.status.ACNET_NO_SUCH
-        if exception != acsys.status.ACNET_UTIME:
-            raise
-
+    except acsys.status.AcnetRequestTimeOutQueuedAtDestination:
+        raise acsys.status.AcnetNoSuchRequestOrReply()
+    except acsys.status.AcnetUserGeneratedNetworkTimeout:
         return None
+    except acsys.status.Status:
+        raise
```

Comment:
I think these lines go away. If the previous two lines didn't match the exception, it'll get re-raised so these lines don't have to do it.